core/core_modules/help.js

```diff
@@ -2,82 +2,81 @@ var figlet = require.safe('figlet'),
+    this.packageInfo.homepage +  '\n\n';
 
-	help = constructHelpMessage(help, this.coreModules, context);
+    help = constructHelpMessage(help, this.coreModules, context);
     return constructHelpMessage(help, this.loadedModules, context);
 },
 
 longDescription = function(moduleName, context) {
-	var module = checkIfModuleExists(this.coreModules, moduleName);
+    var module = checkIfModuleExists(this.coreModules, moduleName);
 
-	if (!module || module.length === 0) {
-		// Check loaded modules, as commnd not in core modules
-		module = checkIfModuleExists(this.loadedModules, moduleName);
-	}
+    if (!module || module.length === 0) {
+        // Check loaded modules, as commnd not in core modules
+        module = checkIfModuleExists(this.loadedModules, moduleName);
+    }
 
-	if (!module || module.length === 0) {
-		return 'Cannot provide help on module that was not found. Has it been disabled?';
-	}
+    if (!module || module.length === 0) {
+        return 'Cannot provide help on module that was not found. Has it been disabled?';
+    }
 
-	if (module.length > 1) {
-		return 'More than one module has the same name. Please fix this before continuing.';
-	}
+    if (module.length > 1) {
+        return 'More than one module has the same name. Please fix this before continuing.';
+    }
 
-	var help = '',
-        cmdHelp = module.ignoreHelpContext
-            ? module.help(context.commandPrefix)
-            : module.help.call(context, context.commandPrefix);
+    var help = '',
+        cmdHelp = module.ignoreHelpContext ?
+        module.help(context.commandPrefix) :
+        module.help.call(context, context.commandPrefix);
 
-	for (var i = 0; i < cmdHelp.length; i++) {
-		var text = cmdHelp[i].length === 3 ? cmdHelp[i][2] : cmdHelp[i][1];
-		help += cmdHelp[i][0] + '\n--------------------\n' + text + '\n\n';
-	}
-	return help;
+    for (var i = 0; i < cmdHelp.length; i++) {
+        var text = cmdHelp[i].length === 3 ? cmdHelp[i][2] : cmdHelp[i][1];
+        help += cmdHelp[i][0] + '\n--------------------\n' + text + '\n\n';
+    }
+    return help;
 };
 
 exports.match = function(event, commandPrefix) {
-    return event.arguments[0] === commandPrefix + exports.platform.packageInfo.name
-        || event.arguments[0] === commandPrefix + 'help';
+    return event.arguments[0] === commandPrefix + exports.platform.packageInfo.name || event.arguments[0] === commandPrefix + 'help';
 };
 
 exports.run = function(api, event) {
-	var commands = event.arguments,
+    var commands = event.arguments,
         context = {
             commandPrefix: api.commandPrefix
         },
-		help;
-	if (commands.length === 1) {
-		help = shortSummary.call(exports.platform, context);
-	}
-	else {
-		commands.splice(0, 1);
-		help = longDescription.call(exports.platform, commands.join(' '), context);
-	}
+        help;
+        
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://www.codacy.com/app/theprojectshop/Kassy/file/2650226390/issues/source?bid=3282987&fileBranchId=3291254#l67)